cmd/docker/docker.go

```diff
@@ -298,6 +303,15 @@ func runDocker(dockerCli *command.DockerCli) error {
 		return err
 	}
 
+	mp := dockerCli.MeterProvider(ctx)
+	defer mp.Shutdown(ctx)
+	// TODO(krissetto): get the package name and version properly for instantiating the meter
+	meter := mp.Meter(
+		"github.com/docker/cli",
+		metric.WithInstrumentationVersion(cmd.Version),
+	)
```

Comment:
As mentioned in another place, I'd recommend moving this to a helper function and to avoid passing around the meter.